src/libos/Makefile

```diff
@@ -127,22 +132,28 @@ ifneq ($(OCCLUM_COV),)
         -Coverflow-checks=off -Cpanic=abort"
 endif
 
+ifeq ($(OCCLUM_UT), 1)
+    LIBOS_FEATURES += unit_test
+endif
+ifneq ($(OCCLUM_RELEASE_BUILD), 1)
+    LIBOS_FEATURES += unit_test
+endif
+
+$(LIBOS_CORE_RS_A): $(RUST_SRCS)
 # Release build is for production use. We enable code coverage only for debug
-# build.  It also simplifies the implementation as the release and debug build
+# build. It also simplifies the implementation as the release and debug build
 # have different output paths.
 ifeq ($(OCCLUM_RELEASE_BUILD), 1)
-$(LIBOS_CORE_RS_A): $(RUST_SRCS)
 	@RUSTC_BOOTSTRAP=1 RUSTC_WRAPPER=$(RUSTC_WRAPPER) cargo build --release --target-dir=$(RUST_TARGET_DIR) -Z unstable-options --out-dir=$(RUST_OUT_DIR) --features "$(LIBOS_FEATURES)"
 	@echo "CARGO (release) => $@"
 else ifneq ($(OCCLUM_COV),)
-$(LIBOS_CORE_RS_A): $(RUST_SRCS)
 	@CARGO_INCREMENTAL=0 RUSTC_BOOTSTRAP=1 RUSTFLAGS=$(COV_FLAGS) cargo build --target-dir=$(RUST_TARGET_DIR) -Z unstable-options --out-dir=$(RUST_OUT_DIR) --features "$(LIBOS_FEATURES)"
-	@echo "CARGO (debug + cov) => $@"
```

Comment:
I suppose we should keep ` + cov`.

Reply:
I output all the features including cov and ut in the `Makefile`. Should we output `cov` here alone?